godot-bindings/src/godot_version.rs

```diff
@@ -6,28 +6,11 @@
 
 //#![allow(unused_variables, dead_code)]
 
+use crate::GodotVersion;
 use regex::{Captures, Regex};
 use std::error::Error;
 use std::str::FromStr;
 
-#[derive(Debug, Eq, PartialEq)]
-pub struct GodotVersion {
-    /// the original string (trimmed, stripped of text around)
-    pub full_string: String,
-
-    pub major: u8,
-    pub minor: u8,
-
-    /// 0 if none
-    pub patch: u8,
-
-    /// alpha|beta|dev|stable
-    pub status: String,
-
-    /// Git revision 'custom_build.{rev}' or '{official}.rev', if available
-    pub custom_rev: Option<String>,
-}
-
```

Comment:
Because otherwise we'd need to pull in `regex` as a dependency even when it's not needed. Since `godot_version` imports `regex`.

Reply:
I see, good reasoning.

Strictly speaking, `regex` is now needed anyway because of 
#[cfg(gdextension_precise_version = "4.0")], but of course this can change again. So let's stay with having it in `lib.rs`. Maybe you could add a short comment...